src/commands/performance/bandwidth_throttle/util.ts

```diff
@@ -0,0 +1,43 @@
+import { Performance, DataFormatter } from '../../../sdk';
+import { OutputFormatter } from '../../../util/formatter';
+
+const outputFormat = {
+	table: [
+		{
+			header: 'Upload Byte Per Second',
+			property: 'upload.bytes_per_second',
+		},
+		{
+			header: 'Upload Windows',
+			property: 'upload.window',
+			transform: val => windowTransform(val)
+		},
+		{
+			header: 'Download Byte Per Second',
+			property: 'download.bytes_per_second',
+		},
+		{
+			header: 'Download Windows',
+			property: 'download.window',
+			transform: val => windowTransform(val)
+		}
+	],
+	json: [
+		'upload',
+		'download'
+	]
+};
```

Comment:
Would be good to see disabled property value for each in table and json. Maybe it's not shown in json if its set to its default?